### PR TITLE
[search] Hide HouseToStreet street identifier type from FeaturesLayerMatcher.

### PR DESCRIPTION
**Pull request by tatiana-yan**

### CODE REVIEW

File: search/reverse_geocoder.hpp [hunk 1]
```diff
@@ -95,6 +95,9 @@ class ReverseGeocoder
   std::pair<std::vector<Street>, uint32_t> GetNearbyFeatureStreets(FeatureType & ft) const;
   /// Same as GetNearbyFeatureStreets but returns streets from MWM only.
   std::pair<std::vector<Street>, uint32_t> GetNearbyOriginalFeatureStreets(FeatureType & ft) const;
+  /// For |houseId| with street information sets |streetId| to FeatureID of street corresponding
```
Comment (mpimenov) [inline search/reverse_geocoder.hpp]: s/corresponding/corresponding to/

File: search/reverse_geocoder.hpp [hunk 2]
```diff
@@ -95,6 +95,9 @@ class ReverseGeocoder
...
+  /// |houseId| and returns true. Returs false otherwise.
+  bool GetFeatureStreetId(FeatureID const & houseId, FeatureID & streetId) const;
```
Comment (mpimenov) [inline search/reverse_geocoder.hpp]: GetStreetByHouse?

**Comment by tatiana-yan** [comment]
JTALL

File: search/reverse_geocoder.hpp (hunk 1 above)
Comment (tatiana-yan) [reply search/reverse_geocoder.hpp]: done

File: search/reverse_geocoder.hpp (hunk 2 above)
Comment (tatiana-yan) [reply search/reverse_geocoder.hpp]: done

**Pull request merged**